api/fossa/locator.go

```diff
@@ -92,6 +106,27 @@ func NormalizeGitURL(project string) string {
 	return noHTTPSPrefix
 }
 
+// NormalizeGitURL normalizes all forms of git remote URLs to a single standard
+// form. This works around the backend only normalizing strings starting with http.
+func NormalizeGitURLTest(project string) string {
+	// Remove fetcher prefix (in case project is derived from splitting a locator on '$').
+	noFetcherPrefix := strings.TrimPrefix(project, "git+")
+
+	// Normalize Git URL format only if not ssh
+	if strings.HasPrefix(noFetcherPrefix, "http") {
+		noFetcherPrefix = strings.TrimSuffix(noFetcherPrefix, ".git")
+	}
+
+	// Will not need this part ever
+	// handleGitHubSSH := strings.Replace(noGitExtension, "git@github.com:", "github.com/", 1)
```

Comment:
I left this comment for reference on this commit. Will remove on the next. This normalizes ssh github urls however because we do not normalize on upload this will direct requests to the wrong locator.